viite-UI/src/view/RoadAddressProjectEditForm.js

```diff
@@ -0,0 +1,212 @@
+(function (root) {
+  root.RoadAddressProjectEditForm = function(projectCollection) {
+    var currentProject = false;
+    var selectedProjectLink = false;
+    var staticField = function(labelText, dataField) {
+      var field;
+      field = '<div class="form-group">' +
+        '<p class="form-control-static asset-log-info">' + labelText + ' : ' + dataField + '</p>' +
+        '</div>';
+      return field;
+    };
+    var actionSelectedField = function() {
+      //TODO: cancel and save buttons Viite-374
```

Comment:
The save button is working, but the cancel still have some issues to solve on VIITE-377 (Form after user has clicked "Peruuta"(Cancel))